lib/stdlib/src/gen_statem.erl

```diff
@@ -346,8 +348,20 @@
     Status :: term() when
       StatusOption :: 'normal' | 'terminate'.
 
+%% Format the callback module status in some sensible that is
+%% often condensed way.
+-callback format_status(Status) -> NewStatus when
+      Status :: #{ state => state(),
+                   data => data(),
+                   reason => term(),
+                   queue => [{event_type(), term()}],
+                   postponed => [{event_type(), term()}],
+                   log => [sys:system_event()] },
```

Comment:
Is not `timeouts => Something,` missing here?

Reply:
Yes, good catch. Missing both here and in the docs. Fixed.